protocol/docker-compose.yml

```diff
@@ -116,7 +116,7 @@ services:
     image: ghcr.io/skip-mev/slinky-sidecar:v0.4.1
     entrypoint: >
       sh -c "slinky-config --chain dydx --node-http-url http://dydxprotocold0:1317 --oracle-config-path /etc/oracle.json &&
-             slinky --oracle-config-path /etc/oracle.json --update-market-config-path /etc/market.json"
+             slinky --oracle-config-path /etc/oracle.json --update-market-config-path /etc/market.json --log-level error"
```

Comment:
Did you mean to include this?

Reply:
Yes.. this already exists on main branch(or a version of it) which can be resolved via a merge conflict resolution

Reply:
Actually, now that I can build without docker, I dont need this anymore, removing